README.md

```diff
@@ -1,96 +1,68 @@
-Overleaf
-========
+<h1 align="center">
+  <br>
+  <a href="https://www.overleaf.com"><img src="https://images.ctfassets.net/nrgyaltdicpt/6gsvc5Ogjmu04I4Miu0uGg/cb1d4391717d2ab8d5e42ede6fb0eef1/overleaf_wide_colour_light_bg.png" alt="Overleaf" width="300"></a>
+</h1>
+
+<h4 align="center">An open-source online real-time collaborative LaTeX editor.</h4>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="https://github.com/overleaf/overleaf/wiki">Wiki</a> •
+  <a href="https://www.sharelatex.com/i/university/onsite">Server Pro</a> •
```

Comment:
This seems to redirect to https://www.overleaf.com/for/enterprises? Are we planning on having a specific page for this?

Reply:
I copied over from other wiki links, didn't realise it was redirecting. Will replace with https://www.overleaf.com/for/enterprises.

Reply:
I think that is the best page to link to at the moment.